plugins/hypervisors/xenserver/test/com/cloud/hypervisor/xenserver/resource/wrapper/xenbase/CitrixRequestWrapperTest.java

```diff
@@ -1320,14 +1322,23 @@ public void testOvsDestroyTunnelCommandFailed() {
 
     @Test
     public void testUpdateHostPasswordCommand() {
-        final UpdateHostPasswordCommand updatePwd = new UpdateHostPasswordCommand("test", "123");
+        final ExecutionResult executionResult = Mockito.mock(ExecutionResult.class);
+
+        final UpdateHostPasswordCommand updatePwd = new UpdateHostPasswordCommand("test", "123", "127.0.0.1");
+
+        final StringBuffer buff = new StringBuffer();
+        buff.append(updatePwd.getUsername());
+        buff.append(' ');
+        buff.append(updatePwd.getNewPassword());
+
+        when(citrixResourceBase.executeInVR(updatePwd.getHostIp(), VRScripts.UPDATE_HOST_PASSWD, buff.toString())).thenReturn(executionResult);
```

Comment:
Why do we need this mock here? 
The test always succeeds as the SshHelper.execute fails with "java.io.IOException: There was a problem while connecting to 127.0.0.1:22"
If I add the below line to test, it always fails
verify(citrixResourceBase, Mockito.atLeastOnce()).executeInVR(Mockito.anyString(),Mockito.anyString(), Mockito.anyString());`

Wanted but not invoked:
`citrixResourceBase.executeInVR(
    <any>,
    <any>,
    <any>
);`
-> at `com.cloud.hypervisor.xenserver.resource.wrapper.xenbase.CitrixRequestWrapperTest.testUpdateHostPasswordCommand(CitrixRequestWrapperTest.java:1341)`

However, there were other interactions with this mock:
-> at `com.cloud.hypervisor.xenserver.resource.wrapper.xenbase.CitrixUpdateHostPasswordCommandWrapper.execute(CitrixUpdateHostPasswordCommandWrapper.java:57)`